rpc/tendermint/core/types/responses_util.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015, 2016 Eris Industries (UK) Ltd.
+// This file is part of Eris-RT
+
+// Eris-RT is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Eris-RT is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Eris-RT.  If not, see <http://www.gnu.org/licenses/>.
+
+package core_types
+
+import (
+	"github.com/eris-ltd/eris-db/core/types"
+)
+
+// UnwrapResultDumpStorage does a deep copy to remove /rpc/tendermint/core/types
+// and expose /core/types instead.  This is largely an artefact to be removed once
+// go-wire and go-rpc are deprecated.
+// This is not an efficient code, especially given Storage can be big.
+func UnwrapResultDumpStorage(result *ResultDumpStorage) (*types.Storage) {
```

Comment:
1. this needs to be tested; I do not like golang, and this is both inefficient, ugly - and probably also wrong
2. I do hope that you see the conflict I try to address here; basically we moved the two rpc's into eris-db; but I want to start opening the path to having a single (versioned) rpc - essentially deprecating /rpc/tendermint;
therefore, I dont want to expose these internal types that will disappear (sooner, rather than later)
-- the deep copy is very inefficient, but right now I want to keep moving forward and solve the problem at the core: reclaim our types properly.
-- reclaiming our types currently is tricky because we depend on go-wire, and I not enough faith in my understanding of go-wire to at this point change the encoding of the types.; Instead I want to move to gRPC+protobuf (eg) then remove this all together.